static/battleship_logo1.svg

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why file is called `battleship_logo1` if we have only one 😄 ?

Reply:
You use underscore (`_`) in the file name but in the second image, you use a dash  (`-`). The convention for naming should be consistent.

Reply:
Remember about adding reviewers and yourself as assignees to the PR next time 😄